filters/src/test/java/io/camunda/zeebe/process/test/filters/logger/RecordStreamLoggerTest.java

```diff
@@ -85,6 +90,62 @@ void testLogMultipleVariables() {
         .contains("nullProperty -> null");
   }
 
+  @Test
+  void testDefaultNoneStartEvent() {
+    final RecordStreamLogger logger = new RecordStreamLogger(null);
+
+    final String result =
+        logger.logRecord(
+            ImmutableRecord.builder()
+                .withRecordType(RecordType.EVENT)
+                .withValueType(ValueType.PROCESS_INSTANCE_CREATION)
+                .withIntent(ProcessInstanceCreationIntent.CREATED)
+                .withKey(123)
+                .withValue(
+                    ImmutableProcessInstanceCreationRecordValue.builder()
+                        .withBpmnProcessId("PROCESS")
+                        .withVersion(1)
+                        .build())
+                .build());
+
+    assertThat(result)
+        .contains(
+            "| EVENT               PROCESS_INSTANCE_CREATION          CREATED                       "
+                + "| (Process id: PROCESS), , (default start)");
```

Comment:
❌I don’t like these extra commas that are being logged here.

Reply:
🤔 Yes I noticed this as well, it became very clear because of the different test setup. However, it was already the case for variables. If there are not variables it would be printed before this PR as:
```
 EVENT               PROCESS_INSTANCE_CREATION          CREATED                       | (Process id: PROCESS),
```
Note the additional `,` in the end.

❓ Should I fix that entirely in this PR?

Reply:
Interesting. I must’ve missed that before. Sure go for the full fix 🚀